requirements.txt

```diff
@@ -0,0 +1,8 @@
+-e git+http://github.com/openaustralia/scraperwiki-python.git@morph_defaults#egg=scraperwiki
```

Comment:
If it comes down to it, we can create another file called *requirements-scraperwiki.txt* that just omits this line. I would drop support for ScraperWiki, but [apod.js](https://github.com/openjck/apod.js) uses their API and it would be a little time-consuming to rewrite it.